app/templates/preprints/provider/preprint-detail.hbs

```diff
@@ -161,4 +161,14 @@
         </div>
     </div>
 
+    {{#bs-modal open=showWarning onHide=(action 'hideWarning') as |modal|}}
+        {{#modal.header}}
+            <div class="warning-header"><i class="fa fa-exclamation-triangle"></i> {{t 'content.warning.header'}}</div>
+        {{/modal.header}}
+        {{#modal.body}}<div class="warning-body">{{t 'content.warning.body'}}</div>{{/modal.body}}
+        {{#modal.footer}}
+            {{#bs-button onClick=(action 'hideWarning') type="info"}}{{t 'content.warning.footer.stay'}}{{/bs-button}}
```

Comment:
Minor: `onClick=(action modal.close)` would be better, so there's only one path out of the modal.